doc/command-line-usage.md

```diff
@@ -0,0 +1,227 @@
+# Using the packge support tool from the command line
+
+There are 2 main uses cases for the package support tool
+
+* Package Maintainers - document and communicate expectations to consumers
+* Package Consumers - understand more about their key dependencies and where they 
+  should invest.
+
+The sections which follow document the usage/options of interest for each use
+case.
+
+## Package Consumers
+
+The support tool helps package conumers review and understand the
+package support information provided by maintainers. See
+[Package-support.md](https://github.com/nodejs/package-maintenance/blob/master/docs/PACKAGE-SUPPORT.md)
+which documents the suggested best practice and the specific format of the
+package support information.
+
+Package consumers use the `show` command to display the support information
+provided by package maintainers.
+
+It is understood that this is only the most basic support that can be provided
+to the consumer. The current focus for the support tool is to provide features
+to encourage/support maintainers to add the package support info to their packages.
+In the future more advanced options can be added to the show command to provide 
+more deduplicated or focussed views of the information available.
+
+The general flow is:
+
+* install node.js (version 10.x or later) and ensure it is on the path
+* npm install the package support tool global (or optionally locally in which
+  case the `bin` directory need to be add the current path)
```

Comment:
why not `npx` instead of globally installing?

Reply:
good point, I'll try that out and update

Reply:
@ljharb updated